assets_manager/models.py

```diff
@@ -58,3 +61,8 @@ def get_full_absolute_url(self):
         domain = Site.objects.get_current().domain
 
         return "http://{0}{1}".format(domain, self.get_absolute_url)
+
+
+@receiver(pre_delete, sender=Asset)
+def asset_delete(sender, instance, **kwargs):
+    instance.file.delete(False)
```

Comment:
@jfunez 

Testou se essa exceção realmente acontece?, realizei o teste e não acontece.

Provavelmente isso acontece para as versões mais antigas do Django, nesta versão o objeto do tipo FileSystemStorage (https://docs.djangoproject.com/en/1.10/ref/files/storage/#the-filesystemstorage-class), garante que não haja exceção, caso não exista o nome do arquivo.

Este tipo é o tipo padrão para um campo do FileField.